docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>"can just get the results off the job"  -&gt; something like "use the <code>results()</code>  method"? as written just sorta feels like a tautology not an instruction.</p><p> </p><p>"polling for the status to be completed" -&gt; "polling until the status is <code>completed</code> "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>fixed</p>